UDV-Core/webpack.config.js

```diff
@@ -22,6 +22,10 @@ module.exports = {
     plugins: [
         definePlugin,
         new webpack.optimize.CommonsChunkPlugin({ name: 'udvcore' }),
+        new webpack.ProvidePlugin({
```

Comment:
Please add a comment saying that this is related to alpaca

Reply:
done